### PR TITLE
Add release notes for 2.0.0

### PR DESCRIPTION
**Pull request by alexcrichton**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -8,6 +8,48 @@ Unreleased.
...
+
+### Added
+
+* Cranelift has gained support for forward-edge CFI on the AArch64 backend.
+  [#3693](https://github.com/bytecodealliance/wasmtime/pull/3693)
+
+* A `--disable-parallel-compilation` CLI flag is now implemented for `wasmtime`.
+  [#4911](https://github.com/bytecodealliance/wasmtime/pull/4911)
+
+* A backend for RISC-V 64 has ben added to Cranelift and Wasmtime.
```
Comment (bjorn3) [inline RELEASES.md:25]: *been

Comment (fitzgen) [inline RELEASES.md:25]: Also maybe say "experimental" or "tier 3" here to set expectations?

**Review by fitzgen — APPROVED** [review]
LGTM with typo fix that bjorn3 pointed out

**Comment by alexcrichton** [comment]
Ah that reminds me to update `docs/stability-tiers.md` as well with the new platforms.

**Pull request merged**